test/unit-test/cellular_config.h

```diff
@@ -54,4 +54,24 @@
  */
 #define CELLULAR_IP_ADDRESS_MAX_SIZE                    ( 64U )
 
-#endif /* CELLULAR_CONFIG_H_ */
+#define TEST
+#ifndef LOOP_FOREVER
+  #ifdef TEST
+    extern int NumLoops;
+    #define LOOP_FOREVER() NumLoops--
+  #else
+    #define LOOP_FOREVER() true
+  #endif
+#endif
+
+/* Macro UNIT_TEST will be defined in test\unit-test\CMakeLists.txt for mock library.*/
+#ifdef UNIT_TEST

```

Comment:
This is a test config file - why do we need this guard?

Reply:
Maybe its naming confuses you. This is for mock library to fake the content of ```CellularContext_t```.
I will change it to the following:
```
/* Macro MOCK_LIB_TEST will be defined in test\unit-test\CMakeLists.txt for mock library.*/
#ifdef MOCK_LIB_TEST
typedef struct CellularContext
{
    int test1;
    int test2;
    int test3;
} CellularContext_t;
#endif
```